test_util.sh

```diff
@@ -134,7 +134,7 @@ function run_test() {
 
 	if [ $result -ne $expected ] ; then
 		let "FAILED += 1"
-		test $expected -eq 0 && comment="Expected success, but failed ($result)." || comment="Expected failure ($expected), but succeeded"
+		test $expected -eq 0 && comment="Expected success, but failed ($result)." || comment="Expected failure ($expected), but succeeded ($result)"
```

Comment:
This is definitely not a change from master. When writing this, I was assuming that `$result` would be always zero in the else branch. However, we might expect failure with a different error id. So, while I accept this change, I ask you to file this for master first.
```suggestion
		test $expected -eq 0 && comment="Expected success, but failed ($result)." || comment="Expected failure ($expected), but got $result"
```

Reply:
added in #120